application/src/main/asciidoc/index.adoc

```diff
@@ -0,0 +1,151 @@
+= Vert.x Application Launcher
+:toc: left
+
+In essence, launching a Vert.x application is not a complex task: write a class with a `main` method, create a Vert.x instance and deploy the main verticle.
```

Comment:
we should also mention that this is a replacement for the old launcher / CLI stuff in the intro too